Makefile

```diff
@@ -0,0 +1,24 @@
+# Image can be overidden with an env var.
+DOCKER_IMAGE?=microscaling/microscaling
+
+# Version tag from the code.
+VERSION=`cat VERSION`
```

Comment:
The build target now appends the SHA for development builds. I've added a release target that tags the image as latest and pushes to DockerHub.